### PR TITLE
Release 4.2.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.2.2 (2020-08-31)
+
+- Fixed the overlay service to allow for overlays which cover the base element. [#199](https://github.com/blackbaud/skyux-core/pull/199)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md:3]: I would lean toward making this an enhancement, added support for the `verticalAlignment` setting when the `placement` is set to `'above'` or `'below'`.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.2.2](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.2.2)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Updated

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.2.2](https://developer.blackbaud.com/skyux-core?_sp.spa=pr-release-4.2.2)

**Pull request merged**